test/testautomation_stdlib.c

```diff
@@ -1188,6 +1188,44 @@ stdlib_iconv(void *arg)
     return TEST_COMPLETED;
 }
 
+
+static int
+stdlib_strpbrk(void *arg)
+{
+    struct {
+        const char *input;
+        const char *accept;
+        int expected[2]; /* negative if NULL */
+    } test_cases[] = {
+        { "",               "",             { -1, -1 } },
+        { "abc",            "",             {-1, -1 } },
+        { "Abc",            "a",            {-1, -1 } },
+        { "abc",            "a",            { 0, -1 } },
+        { "a;b;c",          ";",            { 1, 3 } },
+        { "a;b;c",          ",",            { -1, -1 } },
+        { "a:b;c",          ";:",           { 1, 3 } },
+        { "Hello\tSDL\n",   " \r\n\t",      { 5, 9 } },
```

Comment:
Carrying over my suggestion: I think it would be beneficial to have a test case where the character that is found isn't at the last position in the breakset/accept list, as that would account for loops over characters in accept() that forget to break; (or similar) when a match is found, and end up mistakenly overwriting the match boolean with latter characters and fail to detect a match.

Reply:
I think I have some cases with the break character not the last one in breakset.